### ISSUE TITLE
All but "Successfully turned down fire dice" missing from results report after round-ending move.

### ISSUE THREAD

**Issue by jimmosk** [buttonmen-dev/buttonmen#1589]
In http://dev.buttonweavers.com/ui/game.html?game=1574 on the dev site I just won a round with an attack that used a Fire die. I made the attack, was presented with the "turn down Fire dice" screen as expected, did as instructed, and since that move ended the round (leaving neither of us with attacks), we both autopassed and the next round began. So far, as expected. But the green-text summary of the last move that appeared at the top of the screen it took me to just said "Successfully turned down fire dice", and none of the rest of the results -- us passing, the round being over, etc.

I think that's a bug.
![screen shot 2015-04-05 at 3 57 28 pm](https://cloud.githubusercontent.com/assets/6092544/6998349/3b1d0a92-dbad-11e4-9e7f-7e5c08512f71.jpg)

**Comment by blackshadowshade** [comment]
I'll try to address this after #1596 has been approved.

**Comment by blackshadowshade** [comment]
I missed this before, but this is a duplicate of #1489.
